nilearn/plotting/surf_plotting.py

```diff
@@ -952,6 +980,9 @@ def plot_surf(
 
     nilearn.surface.vol_to_surf : For info on the generation of surfaces.
     """
+    if view is None:
+        view = "dorsal" if hemi == "both" else "lateral"
```

Comment:
This choice sounds non-intuitive to me, but that's not a big issue.

Reply:
Yeah my thinking was just that to show both hemispheres somehow (so not left or right), but from then on it could be anything dorsal, ventral, anterior or posterior

Reply:
Should be documented somewhere ?

Reply:
Yes it is:
![Screenshot 2024-12-30 at 6 09 39 PM](https://github.com/user-attachments/assets/db799ae3-f870-4724-b845-f087f2e2627c)